### DISCUSSION TITLE
Does SP handle text case conversion (e.g. lowercase, uppercase...) ? 🙏

### DISCUSSION THREAD

**Discussion by Cicorione** [speedata/publisher#509] [Q&A] [answered]
Dear @pgundlach 

I couldn't find a specific command to change the case on elements like paragraphs or tables, I found that something is mentioned for `DefineMatter`; do exist commands or rules to modify the case of text and sentences?

Thanks... 🙏

**Comment by pgundlach** [comment, marked as answer]
You are aware of `upper-case()` and `lower-case()` xpath functions? See https://doc.speedata.de/publisher/en/xpathfunctions/ and https://github.com/speedata/examples/tree/master/aesopgnatbull for a use case.

Does that help?

**Reply by Cicorione** [reply]
Oh My... What a dumb! 🤦‍♂️

I have been looking all the time for `uppercase` and `lowercase` without dash...
